python/salpytools/salpylib.py

```diff
@@ -283,8 +285,10 @@ def reply_to_transition(self, cmdId):
             LOGGER.info(
                 "WARNING: INVALID TRANSITION from {} --> {}".format(self.State.current_state,
                                                                     self.next_state))
-            self.State.send_logEvent('rejectedCommand', rejected_state=self.COMMAND)
-
+            # Send the ACK
+            self.mgr_ackCommand(cmdId, self.SAL__CMD_NOPERM, 0, "INVALID TRANSITION")
+            # The rejectCommand seems to be deprectaded.
```

Comment:
There doesn't seem to be any reason to keep this code, even commented out.

Reply:
I think you are right, but Michael still wanted to discuss this next Tuesday. I'll remove it anyway. We can also resurrect it later.